sprint/start_clean.sh

```diff
@@ -44,11 +44,8 @@ done
 ddev config --docroot drupal8 --projectname sprint-[ts] --projecttype drupal8
 
 ddev start
-ddev exec git fetch
-ddev exec git reset --hard origin/8.7.x
-ddev exec composer install
-ddev exec drush si standard --account-pass=admin --db-url=mysql://db:db@db:3306/db --site-name="Drupal Sprinting"
-ddev exec drush cr
+ddev exec bash -c 'git fetch && git reset --hard origin/8.7.x && composer install && drush si standard --account-pass=admin --db-url=mysql://db:db@db/db --site-name="Drupal Sprinting" && drush cr'
```

Comment:
#77 would probably be important since this is long-running, but not a blocker for this PR.

Reply:
Shoot, thought that was in. This particular line actually generates enough output that people will understand that things are going on. But I think an explanatory line would be good here.